queries/org/highlights.scm

```diff
@@ -1,25 +1,28 @@
-((timestamp) @timestamp (#match? @timestamp "^\\<.*$")) @PreProc
-((timestamp) @timestamp_inactive (#match? @timestamp_inactive "^\\[.*$")) @Comment
+((timestamp) @timestamp (#match? @timestamp "^\\<.*$")) @OrgTSTimestampActive
+((timestamp) @timestamp_inactive (#match? @timestamp_inactive "^\\[.*$")) @OrgTSTimestampInactive
 ; ((markup) @markup_bold (#match? @markup_bold "^\\s*\\*.*\\*$")) @text.strong
 ; ((markup) @markup_italic (#match? @markup_italic "^\\s*\\/.*\\/$")) @text.emphasis
 ; ((markup) @markup_underline (#match? @markup_underline "^\\s*_.*_$")) @TSUnderline
 ; ((markup) @markup_code (#match? @markup_code "^\\s*\\~.*\\~$")) @String
 ; ((markup) @markup_verbatim (#match? @markup_verbatim "^\\s*\\=.*\\=$")) @String
 ; ((markup) @markup_strike (#match? @markup_strike "^\\s*\\+.*\\+$")) @text.strike
-(headline (stars) @stars (#eq? @stars "*")) @OrgHeadlineLevel1
-(headline (stars) @stars (#eq? @stars "**")) @OrgHeadlineLevel2
-(headline (stars) @stars (#eq? @stars "***")) @OrgHeadlineLevel3
-(headline (stars) @stars (#eq? @stars "****")) @OrgHeadlineLevel4
-(headline (stars) @stars (#eq? @stars "*****")) @OrgHeadlineLevel5
-(headline (stars) @stars (#eq? @stars "******")) @OrgHeadlineLevel6
-(headline (stars) @stars (#eq? @stars "*******")) @OrgHeadlineLevel7
-(headline (stars) @stars (#eq? @stars "********")) @OrgHeadlineLevel8
-(bullet) @Identifier
-(checkbox) @PreProc
-(property_drawer) @Constant
-(drawer) @Constant
-(tag) @Function
-(plan) @Constant
-(comment) @Comment
-(directive) @Comment
+(headline (stars) @stars (#eq? @stars "*")) @OrgTSHeadlineLevel1
+(headline (stars) @stars (#eq? @stars "**")) @OrgTSHeadlineLevel2
+(headline (stars) @stars (#eq? @stars "***")) @OrgTSHeadlineLevel3
+(headline (stars) @stars (#eq? @stars "****")) @OrgTSHeadlineLevel4
+(headline (stars) @stars (#eq? @stars "*****")) @OrgTSHeadlineLevel5
+(headline (stars) @stars (#eq? @stars "******")) @OrgTSHeadlineLevel6
+(headline (stars) @stars (#eq? @stars "*******")) @OrgTSHeadlineLevel7
+(headline (stars) @stars (#eq? @stars "********")) @OrgTSHeadlineLevel8
+(bullet) @OrgTSBullet
+(checkbox) @OrgTSCheckbox
+((checkbox) @check (#eq? @check "\[-\]")) @OrgCheckboxHalfChecked
```

Comment:
You are missing `TS` in names here. That's why I wasn't able to override it.

Reply:
Good catch, fixed.